packages/page-accounts/src/modals/Create.tsx

```diff
@@ -262,147 +278,172 @@ function Create ({ className = '', onClose, onStatusChange, seed: propsSeed, typ
       header={t<string>('Add an account via seed')}
       size='large'
     >
-      {address && isConfirmationOpen
-        ? (
-          <CreateConfirmation
-            address={address}
-            derivePath={derivePath}
-            isBusy={isBusy}
-            name={name}
-            pairType={pairType}
-            seed={seed}
-          />
-        )
-        : (
-          <Modal.Content>
-            <Modal.Columns>
-              <Modal.Column>
-                <AddressRow
-                  defaultName={name}
-                  noDefaultNameOpacity
-                  value={isSeedValid ? address : ''}
+      <Modal.Content>
+        <Modal.Columns>
+          <Modal.Column>
+            <AddressRow
+              defaultName={name}
+              fullLength
+              isEditableName={false}
+              noDefaultNameOpacity
+              value={isSeedValid ? address : ''}
+            />
+          </Modal.Column>
+        </Modal.Columns>
+        {step === 1 && <>
+          <Modal.Columns>
+            <Modal.Column>
+              <TextArea
+                help={t<string>('The private key for your account is derived from this seed. This seed must be kept secret as anyone in its possession has access to the funds of this account. If you validate, use the seed of the session account as the "--key" parameter of your node.')}
+                isAction
+                isError={!isSeedValid}
+                isReadOnly={seedType === 'dev'}
+                label={
+                  seedType === 'bip'
+                    ? t<string>('mnemonic seed')
+                    : seedType === 'dev'
+                      ? t<string>('development seed')
+                      : t<string>('seed (hex or string)')
+                }
+                onChange={_onChangeSeed}
+                seed={seed}
+                withLabel
+              >
+                <Dropdown
+                  defaultValue={seedType}
+                  isButton
+                  onChange={_selectSeedType}
+                  options={seedOpt}
                 />
-              </Modal.Column>
-            </Modal.Columns>
+                <CopyButton
+                  className='copyMoved'
+                  isMnemonic
+                  value={seed}
+                />
+              </TextArea>
+            </Modal.Column>
+            <Modal.Column>
+              <p>{t<string>('The secret seed value for this account. Ensure that you keep this in a safe place, with access to the seed you can re-create the account.')}</p>
+            </Modal.Column>
+          </Modal.Columns>
+          <ExternalWarning />
```

Comment:
I would suggest moving this last, it looks a bit out of place especially when advanced is used. (Since it pertains to the overall account, not only the seed)

Would, alongside the move to the last component, also have it available on both steps.